lib/verk/workers_manager.ex

```diff
@@ -99,7 +99,15 @@ defmodule Verk.WorkersManager do
     if workers != 0 do
       case QueueManager.dequeue(state.queue_manager_name, workers) do
         jobs when is_list(jobs) ->
-          for job <- jobs, do: job |> Job.decode! |> start_job(state)
+          Enum.map(jobs, fn job ->
+            try do
+              job |> Job.decode! |> start_job(state)
```

Comment:
But another problem may come having it inside the Dead set: listing the dead jobs won't be as easy as we don't know if they are decodable so Verk Web for example can't show all the jobs etc